src/Autocomplete.js

```diff
@@ -149,6 +245,16 @@ class Autocomplete extends AutocompleteBase {
 	}
 
 	/**
+	 * Removes the CSS class from that current selected element.
+	 * @protected
+	 */
+	unSelectCurrentFocusedItem_() {
```

Comment:
woah! Good job. I did not predict this behavior. I will add a test case to it too. Thx. :)